ddtrace/_trace/tracer.py

```diff
@@ -666,10 +661,8 @@ def _on_span_finish(self, span: Span) -> None:
 
     def _log_compat(self, level, msg):
         """Logs a message for the given level.
-
         Instead, something like this will be printed to stderr:
             No handlers could be found for logger "ddtrace.tracer"
```

Comment:
```suggestion

        Instead, something like this will be printed to stderr:
            No handlers could be found for logger "ddtrace.tracer"

```